config/v1/types_authentication.go

```diff
@@ -69,6 +72,17 @@ type AuthenticationSpec struct {
 	// This allows internal components to transition to use new service account issuer without service distruption.
 	// +optional
 	ServiceAccountIssuer string `json:"serviceAccountIssuer"`
+
+	// OIDCProviders are OIDC identity providers that can issue tokens
+	// for this cluster
+	// Can only be set if "Type" is set to "OIDC".
+	//
+	// At most one provider can be configured.
+	//
+	// +listType=map
+	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=1
```

Comment:
i assume this is temporary if upstream adds multiple? otherwise, what is the point of a slice limited to a singe entry?

Reply:
yep, that's a future preparation